lib/Graph/Graph.cpp

```diff
@@ -1155,6 +1155,30 @@ CmpEQNode *Function::createCmpEQ(llvm::StringRef name, NodeValue LHS,
   return addNode(new CmpEQNode(name, OT, LHS, RHS));
 }
 
+IsNaNNode *Function::createIsNaN(llvm::StringRef name, NodeValue input) {
+  TypeRef OT = getResultTypeOfBooleanOp(*getParent(), input.getType());
+  return addNode(new IsNaNNode(name, OT, input));
+}
+
+Node *Function::createReplaceNaN(llvm::StringRef name, NodeValue input,
+                                 float value) {
+  // Create IsNaN node.
+  auto *INN = createIsNaN(name.str() + ".isNaN", input);
+
+  // Create tensor of the same size as input to pass to a Select node.
+  auto *valueVar = getParent()->createVariable(
```

Comment:
What should I do here given that we're in the middle of migrating everything to placeholders? This is a constant.

Reply:
I think this is fine for now -- we will eventually move all private Variables to Constants, and we don't yet have Constant.